src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -139,7 +133,7 @@ import WwwPage from '#src/components/WwwFrame/WwwPage';
 import { createIntersectionObserver } from '#src/util/observerUtils';
 import { trackExperimentVersion } from '#src/util/experiment/experimentUtils';
 import { FLSS_ORIGIN_LEND_BY_CATEGORY } from '#src/util/flssUtils';
-import borrowerProfileExpMixin, { HOME_BP_MODAL_EXP_KEY } from '#src/plugins/borrower-profile-exp-mixin';
+import borrowerProfileExpMixin from '#src/plugins/borrower-profile-exp-mixin';
```

Comment:
Are remaining methods in mixin used on this page?

Reply:
Yes, methods and data.